cli/command/cluster/create.go

```diff
@@ -21,33 +24,76 @@ func NewCreateCommand(c cli.Interface) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.IntVarP(&opts.workers, "workers", "w", 2, "Initial number of worker nodes")
-	flags.IntVarP(&opts.managers, "managers", "m", 3, "Intial number of manager nodes")
+	flags.IntVarP(&opts.managers, "managers", "m", 3, "Initial number of manager nodes")
```

Comment:
fine by me, moving the generic options to provider specific options. At least that wouldn't lead to side effects due to different default values and to not always perfect mappings between the cli options and the plugin options.
I couldn't test the PR because of the missing template option, but now that you added it I will give it a try.